README.md

```diff
@@ -95,6 +96,8 @@ jobs:
 
 The above means you have to create secrets in GitHub which can be found within your repository within **Settings** and then **Secrets** and also be careful to check the connection string which you copy from Azure SQL as the connection string has this **Password={your_password}** and you will need to supply the correct password for your connection string.
 
+The `server-name` is optional and is there only to provide backward compatibility. It is strongly recommended to put the server name in the connection string. The connection string uses this template: `Server=<servername>; User ID=<user_id>; Password=<password>; Initial Catalog=<database>`. In case the server name is put both in the `server-name` and in the `connection-string`, the server name used will be the one specified in the `server-name` YAML key.

```

Comment:
Not sure if we should explicitly mention backward compatibility thing in readMe
@pulkitaggarwl can you please review readMe changes?

Reply:
Well, we should as otherwise where would be no reason to keen the server-name option in the YAML file, right?

Reply:
@yorek  your suggestion around using server-name from connection string makes sense. 


Couple of questions
a) can the connection string be passed without server-name ? In that case would we need server-name in YML ?
b) I see in case of conflict the preference is given to Server-Name in YML, should it be other way round ?

Reply:
a) yes and yes
b) that was also my idea, but @kanika1894 proposed to have server-name to take precedence to keep backward compatbility, which make sense

Reply:
In that case, server-name is not always optional, its conditional. Should we update the readme suggesting similar to - "there are two places to define server-name - yml & connection string, if we define in both the yml one takes precedence"

Reply:
@yorek can you please make the necessary changes and once done probably @pulkitaggarwl can review the readMe/text updates

Reply:
The readme says that the option `server-name` is optional and that is correct, as it can be omitted (which is the best practice) as the server to connect to can also be specified in the connection string itself. I think we should keep the readme as is, otherwise we may confuse the readers. The `server-name` is just for backward-compatibility and should *not* be used anymore, as mentioned in the readme too.